### PR TITLE
[Pangle]Collect Adapter information

### PR DESCRIPTION
**Pull request by PangleZhangketong**

### CODE REVIEW

File: ThirdPartyAdapters/pangle/pangle/src/main/java/com/google/ads/mediation/pangle/PangleInitializer.java [hunk 1]
```diff
@@ -97,6 +97,9 @@ public void initialize(
             .setChildDirected(PanglePrivacyConfig.getCoppa())
             .setGDPRConsent(PangleMediationAdapter.getGDPRConsent())
             .setDoNotSell(PangleMediationAdapter.getDoNotSell())
+            .setUserData(String.format(
```
Comment (joshh-devrel) [inline ThirdPartyAdapters/pangle/pangle/src/main/java/com/google/ads/mediation/pangle/PangleInitializer.java:100]: @PangleZhangketong 
Does this have any impact on ad serving? Or is it for Pangle to collect information only?

Comment (PangleZhangketong) [reply ThirdPartyAdapters/pangle/pangle/src/main/java/com/google/ads/mediation/pangle/PangleInitializer.java:100]: It has no impact on ad serving, just to collect who the request is from.

Comment (PangleZhangketong) [reply ThirdPartyAdapters/pangle/pangle/src/main/java/com/google/ads/mediation/pangle/PangleInitializer.java:100]: > 

It has no impact on ad serving, just to collect who the request is from.

**Review by thanvirm — COMMENTED** [review]
Thank you for contributing this pull request.

This pull request looks good but it fails Google's internal unit tests. Sorry about that! We will clone this pull request internally, fix the internal unit tests tests and then we will merge the pull request.

**Comment by PangleZhangketong** [comment]
> Thank you for contributing this pull request.
> 
> This pull request looks good but it fails Google's internal unit tests. Sorry about that! We will clone this pull request internally, fix the internal unit tests tests and then we will merge the pull request.

Excuse me, when will this pull request be added, this information is very important to us.

**Pull request closed without merging**